doc/COMPILE.md

```diff
@@ -606,6 +606,27 @@ running cmake like to enable debug symbol.
 For Gentoo it's recommended to emerge sys-apps/lsb-release to name the package
 right even thou not required.
 
+### With Flatpak (Beta)
+
+It is also possible to build a Flatpak package containing Elektra:
+
+```bash
+# install flatpak and flatpak-builder, using the package manager of your distro
+$ apt install flatpak flatpak-builder
```

Comment:
```suggestion
$ apt install flatpak flatpak-builder elfutils
```

otherwise the build fails for me with:

```
Running: mv /app/share/icons/hicolor/scalable/apps/elektra.svg /app/share/icons/hicolor/scalable/apps/org.libelektra.kdb.svg
Running: mv /app/share/applications/org.libelektra.elektra-qt-editor.desktop /app/share/applications/org.libelektra.kdb.desktop
Running: sed -i -e 's/Categories=.*/Categories=System;X-Configuration;/g' /app/share/applications/org.libelektra.kdb.desktop
Running: sed -i -e 's/Icon=.*/Icon=org.libelektra.kdb/g' /app/share/applications/org.libelektra.kdb.desktop
compressing debuginfo in: /home/markus/Projekte/Elektra/libelektra/.flatpak-builder/rofiles/rofiles-ktTIPR/files/bin/kdb
Warning: eu-elfcompress not installed, will not compress debuginfo
stripping /home/markus/Projekte/Elektra/libelektra/.flatpak-builder/rofiles/rofiles-ktTIPR/files/bin/kdb to /home/markus/Projekte/Elektra/libelektra/.flatpak-builder/rofiles/rofiles-ktTIPR/files/lib/debug/bin/kdb.debug
Error: module libelektra: Kindprozess »eu-strip« konnte nicht ausgeführt werden (Datei oder Verzeichnis nicht gefunden)
```

Reply:
Thanks for the comment! I have added it to the docs.